packages/components/src/Combobox/components/ComboboxActivator/ComboboxActivator.test.tsx

```diff
@@ -61,20 +62,25 @@ describe("ComboboxActivator", () => {
     expect(activator).toBeInTheDocument();
   });
 
-  it("renders without 'combobox' role if children are not a Chip or Button", () => {
+  it("renders provided element with 'combobox' role if children are not a Chip or Button", () => {
     const { getByText, queryByRole } = renderComboboxActivator(
-      <div>Teammates</div>,
+      (api: ComboboxCustomActivatorProps) => (
+        <div role={api.role}>Teammates</div>
+      ),
       true,
     );
     expect(getByText("Teammates")).toBeInTheDocument();
-    expect(queryByRole("combobox")).not.toBeInTheDocument();
+    expect(queryByRole("combobox")).toBeInTheDocument();
   });
 });
 
-function renderComboboxActivator(child: ReactElement, open: boolean) {
+function renderComboboxActivator(
+  child: ReactElement | ((args: ComboboxCustomActivatorProps) => JSX.Element),
+  open: boolean,
+) {
   return render(
     <ComboboxContextProvider
-      setOpen={setOpen}
+      toggleOpen={toggleOpen}
       handleClose={handleClose}
       selectionHandler={jest.fn()}
```

Comment:
Just a comment, I don't think `handleClose` needs to be mocked anymore. Looking at the tests we could remove change line 84 to be `handleClose={jest.fn()}` but this isn't blocking

Reply:
you're right. at the Activator level we are no longer concerned with `handleClose` being called directly. it's still exposed in the provider at all for other needs, but yeah there's no need for it to be a function we're storing in a variable anymore.